### PR TITLE
[TOPI] operator support: logical_and, logical_or, logical_not

### PR DESCRIPTION
**Pull request by cchung100m**
Following the [issue](https://github.com/dmlc/tvm/issues/3919)

### CODE REVIEW

**Comment by kevinthesun** [comment]
@cchung100m Can you take a look at the ci failure?

File: topi/python/topi/math.py [hunk 1]
```diff
@@ -408,3 +416,57 @@ def reinterpret(x, dtype):
         The result.
     """
     return cpp.reinterpret(x, dtype)
+
+
+@tvm.tag_scope(tag=tag.ELEMWISE)
+def logical_and(lhs, rhs):
+    """Compute element-wise logical and of data.
+        Parameters
```
Comment (yongwww) [inline topi/python/topi/math.py]: for the op description part, better to keep similar indentation as the rest.  https://github.com/dmlc/tvm/blob/master/python/tvm/relay/op/tensor.py#L421-L454

**Comment by cchung100m** [comment]
Hi @kevinthesun 

Thanks for the reminder, I am working on it.

File: topi/python/topi/math.py (hunk 1 above)
Comment (gyshi) [reply topi/python/topi/math.py]: logical_not also be needed to add to tvm, because doc has this op

Comment (cchung100m) [reply topi/python/topi/math.py]: Hi @yongwww @gyshi
I complete the suggestion you mentioned, please help to review it, thanks.

**Comment by kevinthesun** [comment]
Thanks!

**Pull request merged**